backend/src/nodes/image_util_nodes.py

```diff
@@ -5,80 +5,98 @@
 
 import cv2
 import numpy as np
+import numpy.typing as npt
 from sanic.log import logger
 
 from .categories import IMAGE_UTILITY
 from .node_base import NodeBase
 from .node_factory import NodeFactory
 from .properties.inputs import *
 from .properties.outputs import *
-from .utils.image_utils import alpha_overlay, calculate_ssim
+from .utils.image_utils import blend_images, calculate_ssim
 from .utils.pil_utils import *
 from .utils.utils import get_h_w_c
 
+ndarray32 = npt.NDArray[np.float32]
 
-@NodeFactory.register("chainner:image:overlay")
-class ImOverlay(NodeBase):
-    """OpenCV transparency overlay node"""
+
+@NodeFactory.register("chainner:image:blend")
+class ImBlend(NodeBase):
+    """Blending mode node"""
 
     def __init__(self):
         """Constructor"""
         super().__init__()
-        self.description = "Overlay transparent images on base image."
+        self.description = """Blends overlay image onto base image using 
+            specified mode and opacities."""
         self.inputs = [
-            ImageInput("Base").with_id(0),
+            ImageInput("Base"),
```

Comment:
Why is the bottom layer (base) above the top layer (overlay)?

Reply:
It's the same way that overlay is. This is something we all probably need to hash out in the discord, as I am ambivalent and can think of reasons for both orderings.